packages/postgres-database/src/simcore_postgres_database/models/users.py

```diff
@@ -95,6 +95,13 @@
     #
     # User Privacy Rules ------------------
     #
+    sa.Column(
+        "privacy_hide_username",
```

Comment:
I'm surprised by this. I would have expected it to have been implemented as a user preference.

Reply:
it is a user preference, right?